bin/gem2rpm

```diff
@@ -45,6 +60,11 @@ end
 srpmdir = nil
 specfile = nil
 if options[:srpm]
+  unless File.exist?(gemfile)
```

Comment:
> => Yes, I will remove it from this PR, and open new separate PR, and I will consider other hundled exceptions.
I pushed the modification for your review.
But I have not pushed for new PR about gemfile valication check, because it will use new created show_message method. So, I like to push it after I passed current PR.